humblegen/src/elm.rs

```diff
@@ -180,6 +181,7 @@ fn render_type_ident(type_ident: &ast::TypeIdent) -> String {
         ast::TypeIdent::BuiltIn(atom) => render_atom(atom),
         ast::TypeIdent::List(inner) => format!("List {}", opt_parens(render_type_ident(inner))),
         ast::TypeIdent::Option(inner) => format!("Maybe {}", opt_parens(render_type_ident(inner))),
+        ast::TypeIdent::Result(_okk, _err) => todo!(),
```

Comment:
I think this could straightforwardly be added as `format!("Result {} {}", opt_parens(render_type_ident(err)), opt_parens(render_type_ident(ok)))` --- note the switched type order.

But it's fine to leave that for the Elm implementor.